docs/lib/restapi/fragments/js/fetch.md

```diff
@@ -70,17 +70,17 @@ API
 Example with async/await:
 
 ```javascript
-async function head() { 
+function head() { 
     const apiName = 'MyApiName';
     const path = '/path';
     const myInit = { // OPTIONAL
         headers: {}, // OPTIONAL
     };
 
-    return await API.head(apiName, path, myInit);
+    return API.head(apiName, path, myInit);
 }
 
-head();
+const response = await head();
```

Comment:
```suggestion
head();
```
base on above feedback

Reply:
Definitely better than wrapping in an anonymous function. Less is more. The writing surrounding this block can describe `head` as returning a response promise. The code sample should stay light.